pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1414,6 +1414,42 @@ self: super: {
   # Testsuite trying to run `which haskeline-examples-Test`
   haskeline_0_8_0_0 = dontCheck super.haskeline_0_8_0_0;
 
+  insert-ordered-containers = super.insert-ordered-containers.override {
+    optics-core = self.optics-core_0_3_0_1;
+    optics-extra = self.optics-extra_0_3;
+  };
+
+  # Tests depend on HTF and it is broken, 2020-08-23
+  list-t = dontCheck super.list-t;
+  superbuffer = dontCheck (doJailbreak super.superbuffer);
+  stm-containers = dontCheck super.stm-containers;
+
+  # Fails with "supports custom headers"
+  Spock-core = dontCheck super.Spock-core;
+
+  # Needed by Hasura  1.3.1
+  dependent-map_0_2_4_0 = super.dependent-map_0_2_4_0.override {
+    dependent-sum = self.dependent-sum_0_4;
+  };
+
+  # Hasura 1.3.1
+  graphql-engine = (super.graphql-engine.override {
```

Comment:
Why don't you configure these overrides in `pkgs/development/haskell-modules/non-hackage-packages.nix` where you define the package?

Reply:
I am sorry, that I didn‘t realize this before. But I agree, that it would be better to do those overrides directly at the point where you you include the modules. Sorry, that I didn‘t know.

Reply:
There is a comment at the top of that file that reads:

```
# This file should only contain packages that are not in ./hackage-packages.nix.
# Attributes in this set should be nothing more than a callPackage call.
# Overrides to these packages should go to either configuration-nix.nix,
# configuration-common.nix or to one of the compiler specific configuration
# files.
```
I assume this is in case it gets to hackage things continue working. Doing the requested changes goes against that comment, should I still do them?

Reply:
Huh, I would leave that one for @peti to decide. But I think the comment makes sense. Imagine hasura being released to hackage without any code changes. Then we would still need the overrides in configuration-common.nix.

Reply:
/cc @basvandijk

Reply:
Copying the author of the comment to clarify - both options seem sensible to me.